lib/src/components/molecules/comment.dart

```diff
@@ -0,0 +1,265 @@
+                            author,
+                            style: ZetaTextStyles.labelLarge.copyWith(
+                              fontWeight: FontWeight.w500,
+                            ),
+                          ),
+                          const Spacer(),
+                          if (timeStamp != null)
+                            Padding(
+                              padding: EdgeInsets.only(left: spacing.small),
+                              child: Text(
+                                timeStamp!,
+                                style: ZetaTextStyles.bodyXSmall.copyWith(color: colors.textSubtle),
+                              ),
+                            ),
+                        ],
+                      ),
+                      Padding(
+                        padding: EdgeInsets.only(top: spacing.small),
+                        child: Text(
+                          comment,
+                          style: Theme.of(context).textTheme.bodyMedium,
+                        ),
+                      ),
+                      if (attachment != null)
+                        Padding(
+                          padding: EdgeInsets.only(top: spacing.large),
+                          child: _AttachmentRow(
+                            attachment: attachment!,
+                            downloadCallback: downloadCallback,
+                            customThumbnail: attachmentThumbnail,
+                          ),
+                        ),
+                    ],
+                  ),
+                ),
+              );
+            },
+          ),
+        ),
+      ],
+    );
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties
+      ..add(StringProperty('comment', comment))
+      ..add(StringProperty('timeStamp', timeStamp))
+      ..add(StringProperty('author', author))
+      ..add(DiagnosticsProperty<bool>('isReply', isReply))
+      ..add(ObjectFlagProperty<VoidCallback?>.has('onDelete', onDelete))
+      ..add(ObjectFlagProperty<VoidCallback?>.has('onReply', onReply))
+      ..add(DiagnosticsProperty<ZdsChatAttachment?>('attachment', attachment))
+      ..add(ObjectFlagProperty<VoidCallback?>.has('downloadCallback', downloadCallback))
+      ..add(StringProperty('deleteSemanticLabel', deleteSemanticLabel))
+      ..add(StringProperty('replySemanticLabel', replySemanticLabel));
+  }
+}
+
+class _AttachmentRow extends StatelessWidget {
+  const _AttachmentRow({
+    required this.attachment,
+    this.customThumbnail,
+    this.downloadCallback,
+  });
+
+  final ZdsChatAttachment attachment;
+  final VoidCallback? downloadCallback;
+  final Widget? customThumbnail;
+
+  @override
+  Widget build(BuildContext context) {
+    final spacing = Zeta.of(context).spacing;
+    final colors = Zeta.of(context).colors;
+    final radius = Zeta.of(context).radius;
+
+    return Material(
+      child: InkWell(
```

Comment:
I guess the design is not clear but i think some padding around the ink well splash would be better 
<img width="299" alt="image" src="https://github.com/user-attachments/assets/031328be-f57e-4ba7-bff5-cba5f3ce834b">